avm/res/compute/image/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,68 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+// e.g., for a module 'network/private-endpoint' you could use 'dep-dev-network.privateendpoints-${serviceShort}-rg'
+param resourceGroupName string = 'dep-${namePrefix}-compute.images-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = 'eastus'
```

Comment:
Will be overwritten. Please refer to: https://github.com/Azure/bicep-registry-modules/pull/792#discussion_r1455072261

Reply:
done